src/screens/DeviceDetailsScreen.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import type { Device, DeviceMutator } from 'brewskey.js-api';
+import type DAOEntityStore from '../stores/DAOEntityStore';
+
+import * as React from 'react';
+import { inject, observer } from 'mobx-react';
+import { Text, View } from 'react-native';
+import flatNavigationParamsAndScreenProps from '../common/flatNavigationParamsAndScreenProps';
+
+type Props = {|
+  deviceStore: DAOEntityStore<Device, DeviceMutator>,
+  id: string,
+  navigation: Object,
```

Comment:
I keep seeing `navigation` all over and I saw a todo. 

I think the best Idea is to `type NavigationType = Object` somewhere so it's easier to fix.